### PR TITLE
update UI for faucet

### PR DESCRIPTION
**Pull request by XiaoYhun**

### CODE REVIEW

File: src/components/Menu/FaucetModal.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ wallet: account, program: rewardData.program }),
+      })
+      const content = await rawResponse.json()
+      if (content) {
+        addPopup({
+          simple: {
+            title: `Request to Faucet - Submitted`,
```
Comment (viet-nv) [inline src/components/Menu/FaucetModal.tsx:72]: Please add `t` here for translation

Comment (viet-nv) [reply src/components/Menu/FaucetModal.tsx:72]:
```
t`Request to Faucet - Submitted`
```

File: src/components/Menu/FaucetModal.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+            success: true,
+            summary: `You will receive ${
```
Comment (viet-nv) [inline src/components/Menu/FaucetModal.tsx:74]: same here, add translation

File: src/components/Menu/FaucetModal.tsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    } catch (error) {
+      console.log(error)
+    }
+  }
+  useEffect(() => {
+    if (!chainId || !account) return
+    const getRewardAmount = async () => {
+      try {
+        const { data } = await fetch(
+          `https://reward.dev.kyberengineering.io/api/v1/faucets?wallet=${account}&chainId=${chainId}`,
```
Comment (viet-nv) [inline src/components/Menu/FaucetModal.tsx:95]: do not hard code API URL like this, please move it to .env

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/components/Menu/FaucetModal.tsx (hunk 3 above)
Comment (XiaoYhun) [reply src/components/Menu/FaucetModal.tsx]: just did this when waiting for last api update

**Review by tiendv89 — APPROVED** [review]

**Review by viet-nv — APPROVED** [review]

**Pull request merged**